src/components/challengeWidget/challengeWidget.js

```diff
@@ -0,0 +1,39 @@
+import { Link } from "@reach/router";
+import React from "react";
+
+import "./challengeWidget.css";
+
+const ChallengeWidget = ({ title, label, linkTo = "#" }) => (
+  <div className="challengeWidget">
+    <Link to={linkTo}>
+      <h1>{title}</h1>
+      <Link to={label} />
```

Comment:
I'm not really sure what you were trying to do here, but this is the source of the error.